### ISSUE TITLE
SVG icons are missing from Events page

### ISSUE THREAD

**Issue by alexroper** [FDPA/fdpa#226]
I'm getting errors like this:

```
GET https://www.fairdistrictspa.com/assets/images/region-icons/western.svg
net::ERR_INSECURE_RESPONSE
```

My guess is this has something to do with the Let's Encrypt SSL cert not working for the www. subdomain, just the bare domain.

**Comment by alexroper** [comment]
Two problems here:

1. Let's Encrypt's cert didn't include the www subdomain.

I used this command to expand the certificate:

    $ sudo certbot certonly --apache --cert-name fairdistrictspa.com -d fairdistrictspa.com,www.fairdistrictspa.com,staging.fairdistrictspa.com

I also had to run this command to reinstall the verify the certificates.

    $ sudo certbot --apache

2. Minimee rewrites relative urls in the css to be absolute paths, but is using the www.fairdistrictsps.org domain instead of fairdistrictsps.org. No idea where it's getting this from. The siteUrl and baseUrl environment variable don't have the www. 

Maybe the only way to fix this is to make relative paths, absolute paths without the domain name (e.g. /assets/images/ instead of ../images/).
